ocs_sample_library_preview/Subscriptions.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from .BaseClient import BaseClient
+from .OMF.Subscription import Subscription
+from .Securable import Securable
+
+
+class Subscriptions(Securable, object):
+    """
+    Client for interacting with OCS Subscriptions
+    This class does not represent the full functionality of the Subscriptions route and is included to allow management of security on the route
+    """
+
+    def __init__(self, client: BaseClient):
+        """
+        Initializes the Subscriptions client
+        :param client: This is the base client that is used to make REST calls
+        """
+        super().__init__(client=client, collection='Subscriptions')
+
+        self.__base_client = client
+
+        self.__setPathAndQueryTemplates()
+
+    def getSubscriptionById(self, namespace_id: str, subscription_id: str) -> Subscription:
+        """
+        Returns the specified Subscription
+        :param namespace_id: The namespace identifier
+        :param subscription_id: The subscription identifier
+        """
+        if namespace_id is None:
+            raise TypeError
+        if subscription_id is None:
+            raise TypeError
+
+        response = self.__base_client.request('get', self.__subscription_path.format(
+            namespace_id=namespace_id, subscription_id=subscription_id))
+        self.__base_client.checkResponse(
+            response, f'Failed to get Subscription, {subscription_id}.')
+
+        result = Subscription.fromJson(response.json())
+        return result
+
+    def getSubscriptions(self, namespace_id: str, query: str = '', skip: int = 0, count: int = 100,
+                  ) -> list[Subscription]:
+        """
+        Returns a list of Subscriptions
+        :param namespace_id: The namespace identifier
+        :param query: An optional query string to search for matching subscriptions.
+        :param skip: An optional parameter representing the zero-based offset of the first subscription to
+        retrieve. If not specified, a default value of 0 is used.
+        :param count: An optional parameter, between 1 and 1000 (inclusive), that represents the
+        maximum number of retrieved Subscriptions. If not specified, the default is 100.
+        """
```

Comment:
count, skip, and query don't appear to do anything. Unlike the users endpoint, though, using them doesn't throw an error. They just appear to not do anything at all to the results

Reply:
I will remove them since they don't do anything. I only checked that there were no errors.